### PR TITLE
Introducing shanecarr1 to class repo

### PR DESCRIPTION
**Pull request by shanecarr1**
This PR introduces shanecarr1. Please review @githubteacher

### CODE REVIEW

File: _pages/shanecarr1.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+githubHandle: shanecarr1
+pageUrl: https://shanecarr1.github.io/
```
Comment (brianamarie) [inline _pages/shanecarr1.yaml]: Hi @shanecarr1! For the tests to pass, try removing the `https://` from the URL, and also deleting the extra line break in line 5. Let me know if you want or need any help. 😄

**Comment by brianamarie** [comment]
@jvjishnu Hi, please stop making changes to other people's files. Sometimes you may see someone from the GitHub training team do that, but we also make sure to leave notes and give plenty of time so the user can merge the pull request themselves.

**Comment by jvjishnu** [comment]
@brianamarie  Alright, I'm sorry, I'm new and some people changed my files before, so I thought thats how things work here. Thanks for pointing it out. :+1:

**Comment by brianamarie** [comment]
@jvjishnu No problem. If you want to keep helping, that's awesome, but please drop a note in first to see if we can help the person make the changes themselves. I'm sorry to hear someone changed your file - although I'm glad that you're sticking around and enjoying this training repository! 😄

**Comment by jvjishnu** [comment]
@brianamarie  I'll keep that in mind and leave notes from now on. Yeah, I'm enjoying the training, the tutorials are very helpful. :smile:

**Comment by hectorsector** [comment]
@shanecarr1 since the URL returns a 404, I'll ago ahead and close this one out, but please feel free to ping us if you'd like any more help!

**Pull request closed without merging**